### PR TITLE
Move all links to electron-userland

### PR DESCRIPTION
**Pull request by fcastilloec**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -4,12 +4,12 @@
...
 **What operating system are you using?**
 
-**Can you compile the [example app](https://github.com/unindented/electron-installer-windows/tree/master/example) successfully?**
+**Can you compile the [example app](https://github.com/electron-userland/electron-installer-windows/tree/master/example) successfully?**
 
 **If not, paste here the output of the following commands:**
 
 ```
-$ git clone https://github.com/unindented/electron-installer-windows.git
+$ git clone https://github.com/electron-userland/electron-installer-windows.git
```
Comment (malept) [inline .github/ISSUE_TEMPLATE.md]: FYI, the `.git` isn't necessary.

File: README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
 
-# electron-installer-windows [![Version](https://img.shields.io/npm/v/electron-installer-windows.svg)](https://www.npmjs.com/package/electron-installer-windows) [![Build Status](https://img.shields.io/travis/unindented/electron-installer-windows.svg)](http://travis-ci.org/unindented/electron-installer-windows) [![Build Status](https://img.shields.io/appveyor/ci/unindented/electron-installer-windows.svg)](https://ci.appveyor.com/project/unindented/electron-installer-windows) [![Dependency Status](https://img.shields.io/gemnasium/unindented/electron-installer-windows.svg)](https://gemnasium.com/unindented/electron-installer-windows)
+# electron-installer-windows [![Version](https://img.shields.io/npm/v/electron-installer-windows.svg)](https://www.npmjs.com/package/electron-installer-windows) [![Build Status](https://img.shields.io/travis/electron-userland/electron-installer-windows.svg?logo=travis)](http://travis-ci.org/electron-userland/electron-installer-windows) [![Build Status](https://img.shields.io/appveyor/ci/electron-userland/electron-installer-windows.svg?logo=appveyor)](https://ci.appveyor.com/project/electron-userland/electron-installer-windows)
```
Comment (malept) [inline README.md]: Linux/macOS Build Status, Windows Build Status vs. two "Build Status" alt tags.

Comment (fcastilloec) [reply README.md]: I copied how `electron-winstaller` module which only differentiates builds based on icons, I'll change the text inside them, it'll be clearer that way

Comment (malept) [reply README.md]: `electron-winstaller`? As far as I can tell, that differentiates based on service name:

https://github.com/electron/windows-installer/blame/736159e2808d553ae049112cbd67f939a647122a/README.md#L3-L4

Comment (fcastilloec) [reply README.md]: The service name is not shown on the rendered README, so even if it's shown as `Windows Build Status` nothing is really shown on the actual README.
I added the labels directly to the image, so it's more readable but it seems it's a little bit too long. I might just remove the `Build Status` and just leave the name of the OSs, everybody knows that the badges are for build status so I don't think that's needed.
Check how it currently looks here and if you agree with removing the words `Build Status` https://github.com/electron-userland/electron-installer-windows/tree/fix-links#electron-installer-windows---

File: README.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
...
+# electron-installer-windows [![Version](https://img.shields.io/npm/v/electron-installer-windows.svg)](https://www.npmjs.com/package/electron-installer-windows) [![Build Status](https://img.shields.io/travis/electron-userland/electron-installer-windows.svg?logo=travis&label=Linux%2FmacOS%20Build%20Status)](http://travis-ci.org/electron-userland/electron-installer-windows) [![Build Status](https://img.shields.io/appveyor/ci/electron-userland/electron-installer-windows.svg?logo=appveyor&label=Windows%20Build%20Status)](https://ci.appveyor.com/project/electron-userland/electron-installer-windows)
```
Comment (malept) [inline README.md]: Oh, I meant the alt tags, not the text in the build status image itself. Having just the icons is fine.

```suggestion
# electron-installer-windows [![Version](https://img.shields.io/npm/v/electron-installer-windows.svg)](https://www.npmjs.com/package/electron-installer-windows) [![Linux/macOS Build Status](https://img.shields.io/travis/electron-userland/electron-installer-windows.svg?logo=travis)](http://travis-ci.org/electron-userland/electron-installer-windows) [![Windows Build Status](https://img.shields.io/appveyor/ci/electron-userland/electron-installer-windows.svg?logo=appveyor)](https://ci.appveyor.com/project/electron-userland/electron-installer-windows)
```

Comment (fcastilloec) [reply README.md]: Oh! that makes more sense

**Comment by fcastilloec** [comment]
@malept for some reason, even if the branch build in Appveyor passes, it's not reflected in Github and it's stuck at pending... don't know why

**Comment by malept** [comment]
Weird, I'll merge anyway.

**Pull request merged**